### DISCUSSION TITLE
File have different content in many places on GitHub.

### DISCUSSION THREAD

**Discussion by nintyfan** [community/community#78482] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When trying to solve conversation (in pull request), old content was displayed. When go to source branch (on my repository), new content is visible. The same on discussion. When click edit file, new content, on view of file, there is old content.
How to force destination repo to update my file?
https://github.com/calamares/calamares-extensions/blob/4412694cac9efd734ec398e396694d19f1a2d5a6/modules/flatpakinfo/ItemFlatpak.cpp

**Comment by ft10101** [comment]
try clearing the browsers cache , then check again the pull request conversation sometimes browsers save cache aggressively, or if you have any chrome extensions that can also lead to this problem, 

first you must confirm the content is present in the source branch of your repository by crosschecking it once. Also check the commit history that changes were made to correct branch. 

Try refreshing the page multiple times, the problem should be solved till now. if the problem is still there try cloning the repository locally and confirm the changes and try again
